### PR TITLE
Function to use Hexadecimal color code in Colormap

### PR DESCRIPTION
**Pull request by namanb009**
Here is a cleaner PR for #216 

This function will allow users to use hexadecimal color throughout FURY instead of rgb.

Type: Enhancement
Improves User usability.

### CODE REVIEW

File: fury/colormap.py [hunk 1]
```diff
@@ -591,3 +591,15 @@ def _generate_next_color():
...
 
     return _generate_next_color()
+
+
+def hex_to_rgb(color):
+    # Converts Hexadecimal color code to rgb()
+    if color[0] == "#":
+        color = color[1:]
+
+    r = int("0x" + color[0: 2], 0) / 256
```
Comment (skoudoro) [inline fury/colormap.py:601]: `/ 255`  instead of  `/ 256`

File: fury/colormap.py [hunk 2]
```diff
@@ -591,3 +591,15 @@ def _generate_next_color():
...
+    g = int("0x" + color[2: 4], 0) / 256
```
Comment (skoudoro) [inline fury/colormap.py:602]: `/ 255`  instead of  `/ 256`

File: fury/colormap.py [hunk 3]
```diff
@@ -591,3 +591,15 @@ def _generate_next_color():
...
+    b = int("0x" + color[4: 6], 0) / 256
```
Comment (skoudoro) [inline fury/colormap.py:603]: `/ 255`  instead of  `/ 256`

File: fury/tests/test_colormap.py [hunk 4]
```diff
@@ -115,3 +115,10 @@ def test_lab2rgb():
     res = np.round(colormap._lab2rgb(lab_color))
     res[res < 0] = 0
     npt.assert_array_almost_equal(res, expected)
+
+
+def test_hex_to_rgb():
+    expected = np.array([0.99609375, 0.99609375, 0.99609375])
```
Comment (skoudoro) [inline fury/tests/test_colormap.py:121]: These values are weird and wrong, it should be 1 as you did before `np.array([1, 1, 1])`

File: fury/colormap.py [hunk 5]
```diff
@@ -591,3 +591,15 @@ def _generate_next_color():
         return [c for i, c in zip(range(nb_colors), _generate_next_color())]
 
     return _generate_next_color()
+
+
+def hex_to_rgb(color):
```
Comment (skoudoro) [inline fury/colormap.py:596]: Can you add the documentation of the function. See [this function as example](https://github.com/fury-gl/fury/blob/master/fury/colormap.py#L208)

File: fury/tests/test_colormap.py [hunk 6]
```diff
@@ -115,3 +115,10 @@ def test_lab2rgb():
...
+    hexcode = "#FFFFFF"
+    res = colormap.hex_to_rgb(hexcode)
+    npt.assert_array_almost_equal(res, expected)
```
Comment (skoudoro) [inline fury/tests/test_colormap.py:124]: Do not forget to add a test when `hexcode = ffffff` (without #)

**Comment by skoudoro** [comment]
Hi @namanb009,

This PR is close to being done. Any chance to address my comment above?

Thank you

File: fury/tests/test_colormap.py (hunk 6 above)
Comment (namanb009) [reply fury/tests/test_colormap.py:125]: Done

File: fury/colormap.py (hunk 5 above)
Comment (namanb009) [reply fury/colormap.py:596]: Done.

File: fury/tests/test_colormap.py (hunk 4 above)
Comment (namanb009) [reply fury/tests/test_colormap.py]: DOne.

File: fury/colormap.py (hunk 2 above)
Comment (namanb009) [reply fury/colormap.py]: DOne.

File: fury/colormap.py (hunk 1 above)
Comment (namanb009) [reply fury/colormap.py]: Done.

**Comment by namanb009** [comment]
Hey @skoudoro 
I'm sorry for the delay, I got busy with term work.

I've made the changes suggested by you.

Thank you for your review.

**Comment by namanb009** [comment]
I think you can close it now.

**Comment by skoudoro** [comment]
Thank you for the update, looks good, merging

**Pull request merged**